bftengine/src/bftengine/KeyExchangeManager.cpp

```diff
@@ -145,10 +145,13 @@ void KeyExchangeManager::loadPublicKeys() {
 void KeyExchangeManager::exchangeTlsKeys(const std::string& type, const SeqNum& bft_sn) {
   auto keys = concord::util::crypto::Crypto::instance().generateECDSAKeyPair(
       concord::util::crypto::KeyFormat::PemFormat, concord::util::crypto::CurveType::secp384r1);
+  bool use_unified_certs = bftEngine::ReplicaConfig::instance().useUnifiedCertificates;
   std::string root_path =
-      bftEngine::ReplicaConfig::instance().certificatesRootPath + "/" + std::to_string(repID_) + "/" + type;
+      (use_unified_certs)
+          ? bftEngine::ReplicaConfig::instance().certificatesRootPath + "/" + std::to_string(repID_)
```

Comment:
Nit: define const std::string base_path = bftEngine::ReplicaConfig::instance().certificatesRootPath;
to make an operator below shorter.